conf/tw-pycon-org.conf

```
@@ -59,9 +32,13 @@ server {
         alias /var/www/letsencrypt;
     }
 
+    root /opt/PyconWeb/;
+
     # Default path
     rewrite "^/$" /2021 redirect;
     rewrite "^(/(en-us|zh-hant)(/.*)?)$" /2021$1 redirect;
+    rewrite "^/(\d+)/$" /$1/zh-hant redirect;
+    rewrite "^\/\d+\/(.*)\/dashboard\/$" https://tw.pycon.org/prs/$2accounts/login/ redirect;
```

`/prs` is defined in this conf. I think you do not have to define the host here (please test it out).

```suggestion
    rewrite "^\/\d+\/(.*)\/dashboard\/$" /prs/$2accounts/login/ redirect;
```